app/controllers/teams/users_controller.rb

```diff
@@ -29,8 +57,17 @@ def user_present_not_current_user(user)
       user.present? && user != current_user
     end
 
+    def user_params
+      params.require(:user).permit(:email)
+    end
+
     def team
       @team ||= Team.includes(:users).find(params[:team_id])
     end
+
+    def valid_email_address?(email)
```

Comment:
**comment:** I like sticking to the convention where a method with a question mark should always return either 'true' or 'false' boolean values, instead of truthy values (in this case a Match object or nil is returned).

I'm going to suggest a related change above, but feel free to reject if you want!